src/custom/pages/Account/styled.tsx

```diff
@@ -658,3 +658,31 @@ export const CardsSpinner = styled(SpinnerLoader)`
     stroke: ${({ theme }) => (theme.darkMode ? theme.text1 : theme.primary1)};
   }
 `
+
+interface TimeProps {
+  date: string | undefined
+}
+
+export const TimeFormatted = ({ date }: TimeProps) => {
```

Comment:
Should we move to an imported util function?

Reply:
🤷 we could but it's only used in 1 place and is a component